### DISCUSSION TITLE
HOW TO USE ORGANIZATIONS IN GITHUB??

### DISCUSSION THREAD

**Discussion by NAVJOT2015** [community/community#125189] [New to GitHub]
I'm new to GitHub and I do not know how to use GitHub Organizations. If anyone can explain this to me or suggest me youtube video for the same will be very appreciated. Thank You!!

**Comment by BJYOTIBRAT** [comment]
You can watch this video for reference: https://youtu.be/dhxTFy19YIs?si=Bki7b_fh4ytZpLOE (GitHub Organization by Colin Veldkamp). Basically, it is used for organizing your stuff in GitHub.

**Comment by davevad93** [comment]
Hi @NAVJOT2015 ,

Here are some guides about Organizations on GitHub:

- [About organizations](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/about-organizations)
- [Creating a new organization from scratch](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/creating-a-new-organization-from-scratch)
- [Accessing an organization](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/accessing-an-organization)
- [Create a GitHub Organization and Add Collaborators](https://developers.sap.com/tutorials/webide-github-creating-org.html)
- [GitHub Organizations tutorial](https://www.youtube.com/watch?v=dhxTFy19YIs)
- [Managing a GitHub Organization: Tools, Tips, and Best Practices](https://www.youtube.com/watch?v=1T4HAPBFbb0)
- [GITHUB Organisation customise for members and non members](https://www.youtube.com/watch?v=_uuH0IlFodA)

**Comment by karthickkumarsm** [comment]
> I'm new to GitHub and I do not know how to use GitHub Organizations. If anyone can explain this to me or suggest me youtube video for the same will be very appreciated. Thank You!!

Welcome to GitHub! GitHub Organizations are a great way to collaborate with others and manage access to your repositories. Here's a brief explanation of GitHub Organizations and how to use them, along with a suggestion for a YouTube video that can help you get started.

### What is a GitHub Organization?

A GitHub Organization is a shared workspace that allows you to collaborate on multiple projects. It can be used by teams, companies, or any group of people working together on repositories. An organization can have multiple members and teams, each with different levels of access and permissions to the repositories within the organization.

### Key Features of GitHub Organizations

1. **Teams and Permissions:**
   - Organizations allow you to create teams with specific permissions for different repositories. This helps manage who can read, write, and administer each repository.

2. **Repositories:**
   - You can create repositories within the organization. These can be public or private, and you can control access on a per-repository basis.

3. **Billing and Plans:**
   - Organizations have their own billing plans, which can include advanced features like GitHub Actions, more storage, and better support options.

### How to Create and Manage an Organization

1. **Creating an Organization:**
   - Go to [GitHub Organizations](https://github.com/account/organizations) and click on "New organization."
   - Follow the prompts to create your organization, including setting a name, billing plan, and primary contact.

2. **Inviting Members:**
   - Once your organization is created, you can invite members by going to the organization’s page, selecting "People," and then "Invite member."

3. **Creating Teams:**
   - Under the organization’s page, go to "Teams" and create teams. You can set permissions and assign members to these teams.

4. **Managing Repositories:**
   - Create repositories within the organization by navigating to the organization’s page and clicking "New repository." Set the repository’s visibility and permissions as needed.

### Recommended YouTube Video

For a visual and detailed walkthrough, I recommend the following YouTube video:
- [GitHub Organizations and Teams Tutorial](https://www.youtube.com/watch?v=yXT1ElMEkW8)

This video provides a comprehensive guide on setting up and managing GitHub Organizations, including how to invite members, create teams, and manage repositories.

By following these steps and watching the recommended video, you'll gain a solid understanding of how to use GitHub Organizations effectively. Happy coding and collaborating!
